pkg/local_workflows/ignore_workflow/config.go

```diff
@@ -83,71 +76,106 @@ func defaultFuncWithValidator(existingValue interface{}, userInterface ui.UserIn
 		return existingValue, nil
 	}
 
-	if !interactive {
-		return "", fmt.Errorf("no value provided and interactive flag is false")
+	return "", nil
+}
+
+func promptIfEmpty(value string, userInterface ui.UserInterface, promptHelp string, prompt string, validator func(interface{}) error) (string, error) {
+	if value != "" {
+		return value, nil
+	}
+
+	helperText := "<p class='prompt-help'>" + promptHelp + "</p>"
+	err := userInterface.Output(helperText)
+	if err != nil {
+		return "", err
+	}
+
+	input, err := userInterface.Input(prompt)
+	if err != nil {
+		return "", err
 	}
 
-	input, err := userInterface.Input(inputDescription)
+	err = userInterface.Output("") // new line between prompts
```

Comment:
Suggestion: This seems a bit wrong, trying to keep the mindset of a view unaware implementation. This clearly relates to a line based output. Why can't the spaces be part of the formatting of the prompt?